pyproject.toml

```diff
@@ -20,15 +19,18 @@ classifiers = [
 ]
 
 [tool.poetry.plugins."sceptre.resolvers"]
-"custom" = "resolver.custom:Custom"
 
 [tool.poetry.dependencies]
 python = ">=3.8,<3.12"
+types-PyYAML = "6.0.12.9"
```

Comment:
where is this used?

Reply:
In the type hints like `ScalarNodes` etc